Makefile

```diff
@@ -197,7 +197,7 @@ unsigned-android: stop pre-build check-style prepare-android-build ## Build an u
 test: | pre-run check-style ## Runs tests
 	@npm test
 
-build-pr: | can-build-pr stop pre-build check-style i18n-extract-ci ## Build a PR from the mattermost-mobile repo
+build-pr: | can-build-pr stop pre-build check-style i18n-checks ## Build a PR from the mattermost-mobile repo
```

Comment:
At this point with  al the builds I'm inclined to remove all checks, styles, i18n, tests, etc.. cause the fastlane script actually checks if the git branch is dirty or not and I would expect that everything merged has been checked & tested.

thoughts?

Reply:
In general, I agree, if the fastlane script already does the checks. I think it is outside the scope of this PR though.